### ISSUE TITLE
Partial download for a multi-file torrent

### ISSUE THREAD

**Issue by atomashpolskiy** [atomashpolskiy/bt#12]
Depends on #11.

One corner case to keep in mind is when torrent data is not perfectly aligned with the size of a piece. In such case some unwanted files might be created on the disk. I wonder if these leftovers might simply be deleted as soon as the download is complete -- this is going to affect the bitfield, as some pieces will be only partially available. Currently Bt client will consider these pieces to be "missing" and will attempt to re-download them.

**Comment by codenamestif** [comment]
Mostly torrent clients just add some specific extension to unwanted files and write a part of the chunk there. Probably there can be some specific flag to clean them up or not after downloading is done. Anyway, it would be nice to have this feature.
